.github/workflows/svg-compare-all-views.yml

```diff
@@ -0,0 +1,88 @@
+name: SVG diff (all views)
+on: workflow_dispatch
```

Comment:
For now, this action can only be triggered by button click on GitHub. Once it's properly tested (and I'm sure it actually gives useful feedback), I would set this to push/pull.